docs/UserGuide.md

```diff
@@ -516,13 +517,37 @@ Examples:
     * Deletes the 2nd appointment in the list of appointments.
     * Then, create a 5-hour appointment named "Contract Signing With Charlie" on 22nd Oct 2022 at 4:30 PM, associated with the first person in the contact list
 
+### Exiting the program : `exit`
+
+Exits the program.
+
+Format: `exit`
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+<div markdown="span" class="alert alert-info">
+:information_source: **Note the Data Files to be Copied:**
+
+There are 2 data files in the data folder, `addressbook.json` and `schedule.json`, both of which need to be copied for a complete transfer of all saved data.
+</div>
+
+**Q**: Can I edit the data manually? <br>
+**A**: Yes, simply open `addressbook.json` to edit Contacts and `schedule.json` to edit Appointments. Note that if you change a Contact's name in `addressbook.json`, you will also need to change it in `schedule.json`.
+<div markdown="span" class="alert alert-warning">
+:rotating_light: **Recommended Practice:**
+
+It is recommended that you create a backup of the entire data folder before you perform any manual edits, as you risk losing information if your edits are incorrect.
+</div>
+
+**Q**: Why is **ALL** my data missing?<br>
+**A**: Your data file is probably corrupted (likely in the JSON structure). Do not enter any command, close the application, and fix your data file!
```

Comment:
i suggest the following format for adjusting the JSON to be more forgiving here in #253. I can edit this part later if we are ok with that implementation

Reply:
Ok nevermind this follwos the default and current implementation. no change needed here